### PR TITLE
create legacy init script in dependence of operatingsystem

### PR DESCRIPTION
**Pull request by toepi**
Improve debian based distribution support.

fixes #35

### CODE REVIEW

File: manifests/install.pp [hunk 1]
```diff
@@ -60,7 +60,7 @@
     file { 'kibana-init-script':
       path    => '/etc/init.d/kibana',
       ensure  => 'file',
-      content => template('kibana/kibana.legacy.service.erb'),
+      content => template("kibana/$::kibana::params::init_script_template"),
```
Comment (jlambert121) [inline manifests/install.pp]: Variables interpolated in strings needs to be written as "blah blah ${var name} blah blah"  (need the curly braces around the variable) to keep puppet happy.

**Comment by jlambert121** [comment]
Thanks!  One minor change if you would.

**Comment by githubgossin** [comment]
doesnt fix #35 unless also template use bin/bash instead of bin/sh, or replace '==' with '=' to avoid 'unexpected operator" error on e.g. ubuntu 14.04
http://stackoverflow.com/questions/1089813/bash-dash-and-string-comparison

**Comment by jlambert121** [comment]
@githubgossin does @toepi correct the things you noticed?

**Comment by githubgossin** [comment]
no sorry, but almost, there are still two lines containing '==' in kibana.legacy.service.maincontent.erb which breaks POSIX compliance, both lines are `if [ "$STATUS" == "$PID" ]`

**Comment by toepi** [comment]
@githubgossin, @jlambert121 But now! Why I not using TDD :(. A test ensure that are no == in the init-script now. Have also fix a lint-warning and rebase hole request to master.

But why run this script on my debian system - with bash and dash?

**Comment by githubgossin** [comment]
@toepi, @jlambert121 works like a charm on ubuntu 14.04 now

**Comment by jlambert121** [comment]
Thanks guys!

**Pull request merged**